### ISSUE TITLE
The default IntelliJ coverage agent (1.0.622) can't be found by default

### ISSUE THREAD

**Issue by cristan** [Kotlin/kotlinx-kover#76]
When I add this to the top level Gradle file:

```
plugins {
    id 'org.jetbrains.kotlinx.kover' version '0.4.2'
}
```

The docs then says: "Once applied, the plugin can be used out of the box without additional configuration." This is not true. It gives the error:

```
> Could not resolve all files for configuration ':IntellijKoverConfig'.
   > Could not find org.jetbrains.intellij.deps:intellij-coverage-agent:1.0.622.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/intellij/deps/intellij-coverage-agent/1.0.622/intellij-coverage-agent-1.0.622.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/intellij/deps/intellij-coverage-agent/1.0.622/intellij-coverage-agent-1.0.622.pom
```

The reason that it can't find intellij-coverage-agent 1.0.622, is because this version isn't available on [maven central](https://mvnrepository.com/artifact/org.jetbrains.intellij.deps/intellij-coverage-agent).

Adding this from the docs helps:
```
kover {
    enabled = true                          // false to disable instrumentation of all test tasks in all modules
    coverageEngine.set(kotlinx.kover.api.CoverageEngine.INTELLIJ) // change instrumentation agent and reporter
    intellijEngineVersion.set('1.0.637')    // change version of IntelliJ agent and reporter
    jacocoEngineVersion.set('0.8.7')        // change version of JaCoCo agent and reporter
    generateReportOnCheck.set(true)         // false to do not execute `koverReport` task before `check` task
}
```

Or simply only specify the latest version which _is_ available in Maven Central:
```
kover {
    intellijEngineVersion.set('1.0.639')
}
```

Please make it so the default version of intellijEngineVersion which is available on Maven Central, so first time users won't be greeted with this error.

**Comment by shanshin** [comment]
Hi,
the version of the IntelliJ agent published in Maven Central is incompatible with the version of Kover `0.4.2`.
In the next release, it will be possible to use the IntelliJ agent, which are now published in Maven Central.

**Comment by shanshin** [comment]
Duplicate #34

**Comment by shanshin** [comment]
Fixed in `0.4.3`
